### DISCUSSION TITLE
Goals / Languages / Projects you are working on

### DISCUSSION THREAD

**Discussion by iason9** [EddieHubCommunity/support#478] [General]
Hi everyone, wanted to start a conversation about our goals

- What are your projects for this end of the year and for the next year? 💯 
- are you considering starting your own business 🏢  / get a full-time job 🧑‍💼  / or study 🔖  for 2021?
- which programming languages and frameworks 👨‍💻  you are working on or would love to learn and use in the future.

**Comment by eddiejaoude** [comment]
Great idea @thejason1 👍 

## 2020

- finish off client work
- create more YT content

## 2021

- work more with the community on open source
- more YT content of course 
- thinking about creating a course

That is off the top of my head, I will have a better think over the festive season 🤓

**Reply by iason9** [reply]
Nice wish you good luck 😃

**Reply by eddiejaoude** [reply]
Thank you 🤓 I need it! 😂

**Reply by emmysteven** [reply]
Keep pushing the bar

**Reply by eddiejaoude** [reply]
Thanks 👍

**Comment by collinsMutai** [comment]
# End year / early 2021
Continue working on my FullStack Nanodegree and maintain the momentum till the end. The learning curve is steep but i'm fully prepared lol
# 2021 and beyond
Get into open source, apply for jobs and mentor other newbies and people wanting to break into tech. :)

**Reply by iason9** [reply]
Great!! wish you the best, but i would suggest starting open source by next year, so that you get the momentum of contributing and in the same time you will have a great GitHub profile which means more opportunities for whatever you want to do next.

**Reply by collinsMutai** [reply]
Thank you. Sounds like a great plan.

**Reply by eddiejaoude** [reply]
Awesome! Keep up the great work 👍

**Comment by Vyvy-vi** [comment]
## 2020 end/ early 2021:

- Get TearDrops to v1.3
- Get skilled with Flask 
- Contribute to OpenSource
- Go through school stuff

**Reply by eddiejaoude** [reply]
Open source FTW 💪

**Comment by mstanciu552** [comment]
## Unfinished
React website for school

## New 
Contribute on Open Source
Learn DevOps, docker, Kubernetes

**Reply by eddiejaoude** [reply]
Open source FTW 💪

**Reply by BOLT04** [reply]
if you didn't catch Kelsey's talk on GitHub universe, I think it's cool to watch to know something related to DevOps, Docker and K8S: https://www.youtube.com/watch?v=yIAa5wHsfw4 😃

**Comment by Akshu-on-github** [comment]
Knocking on wood...

## End of 2020/ Beginning of 2021
- Bring my exams to a satisfactory close
- Worked on quite a few certification courses. Need to complete the remaining and organize the same
- Working on a few projects, need to start on a few more

## Well into 2021
- Create a resume
- Get an internship
- College and Club work, ig

**Reply by iason9** [reply]
Wish you success and good luck

**Reply by Akshu-on-github** [reply]
You too @iason9!

**Reply by iason9** [reply]
Thx!

**Reply by eddiejaoude** [reply]
Sounds great 💪

**Comment by emmysteven** [comment]
## End of 2020 👋🏻
Implement CI/CD on my [pet project](https://github.com/emmysteven/restaurant)
Grow my twitter [account](https://twitter.com/emmysteven_) from 206 to 1000 followers at least
Contribute more to open source

## Beginning of 2021 🤗 
Keep working on my [pet project](https://github.com/emmysteven/restaurant)
Land a tech job
Grow my [Linkedin account](https://linkedin.com/in/emmysteven)
Contribute heavily to open source

**Reply by iason9** [reply]
nice, i see you got active these last months, when did you start programing?

**Reply by emmysteven** [reply]
I started programming since 2010, but it was met with a lot of inconsistency, you know life happens.
However I have been consistent for about 2yrs and counting...

**Reply by iason9** [reply]
The same here i got involved with code since 2012 where i started studying in Univesity but had a lot of personal problems i only started 2 years ago where i got a year of experience with a remote job for a french company.

**Reply by emmysteven** [reply]
I've wanted to have an experience working in a distributed team
In order to make that happen, I started taking open source seriously,
That way I can make all the mistakes and not embarrass myself when I do finally get the opportunity to work remotely in a distributed

**Reply by iason9** [reply]
yeah, it is a good call, but do apply for jobs because you learn so much more when you are doing professional work in a team, and you will do mistakes without a doubt but that's how you learn and evolve..

**Reply by emmysteven** [reply]
I'm actively applying, I was even applying for a job while I was chatting with you 😄

**Reply by emmysteven** [reply]
I have an issue I want to point out to you, 23hrs ago, I opened a [pull request](https://github.com/EddieJaoudeCommunity/support/issues/570) requesting to be added to the community
Up until now I have not been added to the community.

**Reply by iason9** [reply]
Yeah just wait a bit more, Eddie has a lot of work/notifications to respond to and he will get back to you shortly and add you without a doubt.

**Reply by emmysteven** [reply]
I'll patiently wait 😁

**Reply by emmysteven** [reply]
Good news 🎉 
I have been given an invite to join the community!

**Reply by iason9** [reply]
Nice, welcome to the community 😄

**Reply by eddiejaoude** [reply]
Thank you all for waiting patiently and the great community work and supporting each other 👍
